lnrpc/signrpc/signer_server.go

```diff
@@ -310,9 +310,10 @@ func (s *Server) SignOutputRaw(ctx context.Context, in *SignReq) (*SignResp,
 
 		// If a witness script isn't passed, then we can't proceed, as
 		// in the p2wsh case, we can't properly generate the sighash.
+		// For P2WKH we expect the PK script to be sent which the
+		// SignOutputRaw method will use correctly when creating the
```

Comment:
nit: I think you mean "SignOutputRaw method will be used correctly"?

Reply:
No, I mean that SignOutputRaw will handle the PK script correctly, even though it's technically not a PK script. I changed the sentence a bit to make it easier to understand.